apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/eobByPatientIdPaged.json

```diff
@@ -2065,14 +2065,14 @@
         "type" : [ {
           "coding" : [ {
             "system" : "https://bluebutton.cms.gov/resources/codesystem/diagnosis-type",
-            "code" : "external-first",
```

Comment:
I think the changes to this file aren't material to your PR here, right? If so, I'd remove them from the PR.